tutorials/5-docker-intro/index.md

```diff
@@ -34,15 +36,21 @@ As opposed to proper virtualization, containerization is fast:
 
 ### Images
 
-Instead of a virtual disk with a full operating system, a container starts from an **image**. The image contains the files that are specific to the container and different from its Linux host. Think of tracing paper recording only the differences laid on top of the host's file system. These can be very small.
+Instead of a virtual disk with a full operating system, a container starts from an **image**. The image contains the files that are specific to the container and different from its Linux host. Think of tracing paper recording only the differences laid on top of the host's file system. These differences can be very small.
+
+You can either create images locally via a **`Dockerfile`** containing the commands that create the image, or you can use an existing image.
+
+<HighlightBox type="info">
 
-You can either create images locally via a **`Dockerfile`** containing the commands that create the image, or you can use an existing image. There are registries that store and/or reference them, like your regular package manager. [Docker Hub](https://hub.docker.com) is the main one, and your local Docker knows how to download directly from it.
+Some registries store and/or reference them, like your regular package manager. [Docker Hub](https://hub.docker.com) is the main one, and your local Docker knows how to download directly from it.
```

Comment:
I think you changed the meaning. I want to say:

> There exist registries that...

You changed it to

> We all know there are registries and some store and/or reference them.

The problem is that it is not obvious that there would be registries. And storing or referencing them is all they do anyway, some only reference, some do both. So it is not like some registries do nothing, others store/reference and others do more than that.

If changing, I would go with a variation on:

> There exist registries that ...

Reply:
@xavierlepretre feel free to change back. 😉